ci-operator/step-registry/assisted/baremetal/operator/gather/assisted-baremetal-operator-gather-commands.sh

```diff
@@ -25,6 +26,51 @@ function getlogs() {
 # Gather logs regardless of what happens after this
 trap getlogs EXIT
 
+echo '#### Gathering Sos reports from all Nodes'
+
+timeout -s 9 30m ssh "${SSHOPTS[@]}" "root@${IP}" DISCONNECTED="${DISCONNECTED:-}" bash - << "EOFTOP"
+    if [ $(virsh list --name |  tr -s '\n'  | wc -l) > 0 ]
+    then
+      export SOS_BASEDIR="/tmp/artifacts/sos"
+      export HOSTFILE="${SOS_BASEDIR}/virtual_hosts.json"
+      mkdir -p "${SOS_BASEDIR}"
+    
+      export VIRT_NET=$(virsh net-list --name  | grep -v default | tr -d "\n")
+      export TMP_HOSTFILE="/tmp/tmp_hostfile.json"
+
+      echo [] > ${TMP_HOSTFILE}
```

Comment:
```suggestion
      echo "[]" > ${TMP_HOSTFILE}
```
I think it's clearer and not some shell shenanigans